### ISSUE TITLE
[Sourcing][Accessibility]: sorting menu is not accessible from keyboard

### ISSUE THREAD

**Issue by I543348** [SAP/fundamental-ngx#9457]
#### Is this a bug, enhancement, or feature request?
bug

#### Briefly describe your proposal.
1) click on sort and sort any column ascending or descending 
2) now click on that column's menu and try to access options using keyboard
3) user should be able to access the options using keyboard and apply sort

#### Which versions of Angular and Fundamental Library for Angular are affected? (If this is a feature request, use current version.)
angular 15
core / platform:0.38.2

#### If this is a bug, please provide steps for reproducing it.
This is replicable on https://sap.github.io/fundamental-ngx/#/platform/table#sortable 
<img width="1251" alt="Screenshot 2023-02-27 at 9 52 04 PM" src="https://user-images.githubusercontent.com/88182358/221620802-48d4ceae-568d-464b-a5ca-fb0e384379c8.png">
